Frameworks/UIKit/UIBezierPath.mm

```diff
@@ -190,75 +150,76 @@ - (void)addArcWithCenter:(CGPoint)center
               startAngle:(CGFloat)startAngle
                 endAngle:(CGFloat)endAngle
                clockwise:(BOOL)clockwise {
-    CGPathAddArc(_path, NULL, center.x, center.y, radius, startAngle, endAngle, clockwise == FALSE);
+    CGPathAddArc(_workingPath, NULL, center.x, center.y, radius, startAngle, endAngle, clockwise == TRUE);
 }
 
 /**
  @Status Interoperable
 */
 - (void)addLineToPoint:(CGPoint)pt {
-    CGPathAddLineToPoint(_path, getTransform(self), pt.x, pt.y);
+    CGPathAddLineToPoint(_workingPath, NULL, pt.x, pt.y);
 }
 
 /**
  @Status Interoperable
 */
 - (void)moveToPoint:(CGPoint)pt {
-    CGPathMoveToPoint(_path, getTransform(self), pt.x, pt.y);
+    CGPathMoveToPoint(_workingPath, NULL, pt.x, pt.y);
 }
 
 /**
  @Status Interoperable
 */
 - (void)closePath {
-    CGPathCloseSubpath(_path);
+    CGPathCloseSubpath(_workingPath);
 }
 
 /**
  @Status Interoperable
 */
 - (CGPathRef)CGPath {
-    return _path;
+    return CGPathCreateCopy(_workingPath);
```

Comment:
You will want to cast this to id and autorelease it. Right now it's returning a +1.

I guess _that_ is why it uses `NS_RETURNS_INNER_POINTER`!